site/src/Controller/ProjectController.php

```diff
@@ -273,7 +275,7 @@ public function viewAction(
             'contributorsGraph' => $this->makeGraph(
                 $projectMetricsProvider->getContributorHistogram($p, Dates::INTERVAL_YEAR)
             ),
-            'isHostedOnGithub' => GithubRepoCrawler::isGithubRepoUrl($p->getSourceRepo()->getUrl()),
+            'isHostedOnGithub' => preg_match_all(self::GH_REGEX,$p->getSourceRepo()->getUrl()),
```

Comment:
Can you move the regex into the SourceRepo and/or GitSourceRepo class, e.g. as `getHostingSite()`? We don't want to have this check in multiple places.